Makefile

```diff
@@ -56,15 +59,17 @@ distclean: clean
 	docker volume rm -f $(CARGO_CACHE_VOLUME_NAME)
 
 testdata/vmlinux:
-	$(curl) -o $@ https://s3.amazonaws.com/spec.ccfc.min/img/hello/kernel/hello-vmlinux.bin
+	$(curl) -o $@ https://s3.amazonaws.com/spec.ccfc.min/img/quickstart_guide/$(arch)/kernels/vmlinux.bin
 
 testdata/firecracker:
-	$(curl) -o $@ https://github.com/firecracker-microvm/firecracker/releases/download/$(firecracker_version)/firecracker-$(firecracker_version)-x86_64
-	chmod +x $@
+	$(curl) -o testdata/firecracker.tgz ${release_url}
+	tar -xvzf testdata/firecracker.tgz -C ./testdata
+	mv testdata/firecracker-$(firecracker_version)-$(arch) testdata/firecracker
+	chmod +x testdata/firecracker
 
 testdata/jailer:
```

Comment:
How does it get `testdata/jailer-$(firecracker_version)-$(arch)`?

Reply:
It is includes in the testdata/firecracker.tgz.

Reply:
If we are downloading them together in one step does it make sense for these to be separate in the Makefile?

Reply:
Maybe the target could be testdata/binaries?

Reply:
Thanks for @IRCody helps. I move all of them under testdata/binaries